src/index.js

```diff
@@ -2,6 +2,7 @@ import 'assets/styles/main.scss';
 import pick from 'lodash/pick';
 import VideoPlayer from './video-player';
 import createVideoPlayerProfile from './video-player-profile';
+import createPlayer from './player';
```

Comment:
Why are we having two separate methods that does exactly the same?
If we really must, just do `export const videoPlayerWithProfile =  player`

Reply:
there is small difference between them (profile property as optional one) and there is possibility that some people are using it - in this case first we should deprecate it and later after some time just remove it